tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -215,6 +215,30 @@ TfLiteStatus CommitPlan(ErrorReporter* error_reporter, MemoryPlanner* planner,
 
 namespace internal {
 
+// Allocate a TfLiteIntArray and copy the contents of a FlatBuffers Vector
+// into it.
+template <class T>
+TfLiteStatus FlatBufferIntArrayToTfLiteIntArray(
+    SimpleMemoryAllocator* allocator, ErrorReporter* error_reporter,
+    const flatbuffers::Vector<T>* flat_array, TfLiteIntArray** result) {
+  TfLiteIntArray* ret =
+      reinterpret_cast<TfLiteIntArray*>(allocator->AllocateFromTail(
+          TfLiteIntArrayGetSizeInBytes(flat_array->Length()),
+          alignof(TfLiteIntArray)));
+  if (nullptr == ret) {
+    error_reporter->Report(
```

Comment:
`TF_LITE_REPORT_ERROR should be used instead of error_reporter, so log strings can be removed to save space if needed.`

Reply:
Commit 2575e9a changes this line to use `TF_LITE_REPORT_ERROR`.